plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtils.java

```diff
@@ -31,6 +33,10 @@
  */
 public class PlayAssetDeliveryUtils {
 
+  public static final String ASSETPACK_DICTIONARY_TYPE_KEY = "type";
```

Comment:
I would rename these to indicate that they're for exceptions.

Reply:
Thanks! Just renamed these variables.